lib/desktop-notifications.ts

```diff
@@ -1,11 +1,23 @@
 import { v4 as uuidv4 } from 'uuid'
 import * as LRUCache from 'lru-cache'
+import * as os from 'os'
+
+export function supportsNotifications() {
+  if (process.platform !== 'win32') {
+    return false
+  }
+
+  const versionComponents = os.release().split('.')
+  const majorVersion = parseInt(versionComponents[0], 10)
+
+  // Only Windows 10 and newer are supported
+  return majorVersion >= 10
```

Comment:
Not sure we even have to parse here do we?

```suggestion
  // Only Windows 10 and newer are supported
  return process.platform === 'win32' && os.release().startsWith('10.')
```

Reply:
Yup, Windows 11 uses `10.` too, but I wanted to be more "future-proof" assuming future versions of Windows will Just Work™ (am I too naive? 🤣 ).

Maybe I should've reworded this PR's title to: disable `desktop-notifications` on Win 7 and 8 😅